src/pages/events/commands.md

```diff
@@ -44,8 +77,8 @@ The `events:subscribe` command subscribes the current provider to the specified
 
 where:
 
--  `type` specifies the origin of the event. Specify `observer` if the event is emitted by a Commerce observer, or specify `plugin` if the event is emitted by a plugin.
--  `event_name` identifies the event to subscribe. For example: `catalog_product_save_after`.
+*  `type` specifies the origin of the event. Specify `observer` if the event is emitted by a Commerce observer, or specify `plugin` if the event is emitted by a plugin.
+*  `event_name` identifies the event to subscribe. For example: `catalog_product_save_after`.
```

Comment:
After the latest changes, we still can subscribe to any observer events. But can't subscribe to the plugin events as the Plugin class wasn't generated for it, we can only subscribe (enable) plugin events that were in config.php but were disabled.
Maybe we should add this as some kind of notice